truss/remote/baseten/api.py

```diff
@@ -309,14 +313,32 @@ def deploy_chain_atomic(
             ]
         )
 
+        # Build the query parameters
+        params = []
```

Comment:
we don't need to change all of this now, we were inheriting this, but this is starting to be a mess. Good reason to move away from graphql for these paths (that aren't even used in the frontend)